docs/docs/general/02-Getting Started/writing-your-first-test.md

```diff
@@ -117,3 +148,7 @@ You can also use `npx jest` to test all your Jest test files and `npx jest <fold
 If your jest config file e.g. `jest.config.ts` is not in your root folder you can use `--config` to specify the location of your config file. In this example the config file is in the `test` folder, that's why we need to provide the location our config file.
 
 You should now see the test suite being executed inside the shell and, actually, your cursor should move to some text shown on your screen and click on that text. :tada: Congratulations! You just executed your first test suite using askui.
+
+### Troubleshooting
+
+If you are having problems with the test execution, [have a look at our Troubleshooting page](../07-Troubleshooting/index.md)
```

Comment:
What is the reason for linking to Troubleshooting again?

Reply:
I want the user to know where they should go when they have problems without searching for it. So putting it on the same page was logical for me.

For example: When I worked through the Getting Started I ran into the MacOS permission problem and did not end up at the Troubleshooting page. It was not clear to me that I had to go there to resolve my issue.